README.md

```diff
@@ -26,7 +26,7 @@ golint prints out style mistakes.
 
 Golint differs from govet. Govet is concerned with correctness, whereas
 golint is concerned with coding style. Golint is in use at Google, and it
-seeks to match the accepted style of the open source Go project.
+seeks to match the accepted style of the open source Go project, [as described here](https://github.com/golang/go/wiki/CodeReviewComments).
```

Comment:
That's not particularly accurate. That page isn't technically a style guide, but I think this new sentence implies that it is.

I'd be fine having this link in here (but as golang.org/wiki/CodeReviewComments), but it needs different prose.